docs/tempo/website/setup/_index.md

```diff
@@ -1,40 +1,38 @@
 ---
-title: Set up a Tempo cluster
-menuTitle: Set up a Tempo cluster
-description: Learn how to set up a Tempo cluster and visualize data
+title: Set up a Tempo server or cluster
+menuTitle: Set up a Tempo server or cluster
+description: Learn how to set up a Tempo server or cluster and visualize data
 weight: 150
 ---
 
-# Set up a Tempo cluster
+# Set up a Tempo server or cluster
 
 Tempo is available as a pre-compiled binary, a Docker image, and as common OS-specific packaging.
 
-No matter which option you choose, setting up your Tempo cluster involves: 
+You can choose to install Tempo on a single system or as a cluster.
 
-1. Naming your cluster
-1. Deploying your cluster
-1. Testing your cluster
-
-This page highlights these steps; more detailed instructions are available on the procedures for deploying clusters.
+This page highlights these steps; more detailed instructions are available on the procedures for installing Tempo.
 
 ## Name your cluster
 
-No matter which deployment method you choose, you need to decide what to call your Tempo cluster.
+If you install Tempo in a cluster, you need to decide what to call it.
 A cluster name must meet the following criteria:
 
 - is 3 to 63 characters long
 - contains lowercase letters, numbers, underscores (_), or hyphens (-)
 - begins with a letter or number
 - ends with a letter or number
 
-## Deploy your cluster
+## Deploy Tempo
 
-Choose a method to deploy your Tempo cluster:
+Choose a method to deploy Tempo:
 
-<!-- - [Deploy on Linux]({{< relref "linux">}}) -->
+- [Deploy on Linux]({{< relref "linux">}})
 - [Deploy on Kubernetes using Tanka]({{< relref "tanka">}})
 
-## Test your cluster
+## Test your installation
 
-Once your cluster is deployed, you can test your cluster by visualizing the traces data with a simple TNS app. 
+Once Tempo is deployed, you can test your cluster by visualizing the traces data with a simple TNS app.
```

Comment:
we should point people away from TNS. we've moved to this project and it is much simpler to setup and produces better traces:

https://github.com/grafana/tempo/blob/main/example/docker-compose/gcs/docker-compose.yaml#L25

Reply:
@hedss  FYI.

Reply:
We should create a getting started using that example as a set up. We have an example app set up using the TNS app. I'll create a GH issue for this so we can make that happen.

Reply:
@hedss and I added content for the Docker example you pointed to.